src/vs/workbench/common/editor/editorGroupModel.ts

```diff
@@ -712,14 +712,17 @@ export class EditorGroupModel extends Disposable implements IEditorGroupModel {
 		this._onDidModelChange.fire(event);
 	}
 
-	public get selectedEditors(): EditorInput[] {
+	get selectedEditors(): EditorInput[] {
 		// Return selected editors in sequential order
-		return this.editors.filter(editor => this.isSelected(editor));
+		return this.editors.filter(editor => this.isSelected(editor)); // TODO I would have assumed `this.selection` to be in sequential order
```

Comment:
@benibenj since `editors` is sequential, maybe `selection` could be too?

Reply:
We would not be able to tell which is the active editor then.

Reply:
Yeah this would require us to keep track of `active` editor again in a separate field and then have `selection` be an array of sequentially selected tabs.